src/test/java/org/candlepin/subscriptions/event/EventControllerTest.java

```diff
@@ -162,6 +167,16 @@ void setup() {
                    "service_type": "OpenShift Cluster"
                  }
                 """;
+    cleanUpEvent =
+        """
+                {
+                   "org_id": "7",
+                   "timestamp": "2023-05-02T00:00:00Z",
+                   "event_type": "snapshot_redhat.com:openshift_dedicated:cluster_hour",
+                   "event_source": "prometheus",
+                   "action": "cleanup"
```

Comment:
This is the new field "action". Note that in the previous events, there is no "action" property because it's intentionally optional.